matching-notification-api/src/main/java/org/phenotips/matchingnotification/finder/internal/LocalMatchFinder.java

```diff
@@ -54,37 +54,32 @@ public int getPriority()
     }
 
     @Override
-    public String getName()
+    public List<PatientMatch> findMatches(List<Patient> patients, Set<String> serverIds,
+        boolean onlyUpdatedAfterLastRun)
```

Comment:
There should be a check that throws an exception if there is an unsupported `serverId` in the set of requested `serverIds`

Reply:
By design each `MatchFinder` selects server Ids that is supports and search matches on each of them.